controllers/nc-news.articles.controller.js

```diff
@@ -0,0 +1,31 @@
+const models = {};
+models.articles = require('../models/nc-news.articles.model.js');
+models.topics = require('../models/nc-news.topics.model.js');
```

Comment:
Really like this pattern where you name where the functions come from. Consider moving this into an `index.js` file in the models folder so you don't have to repeat it anywhere else in the project.